ONSdigital/design-system
# Revert details component to JS version to fix printing issue

### What is the context of this PR?
Fixes: #2869 

This reverts the details component to use the previous JS implementation. This allows us to fix an issue of details components not showing the contents when printed. It will mean that users that have JS turned off will now just see the title and contents as a heading and paragraph and not have a functioning details component.

Because we have switched back to the JS implementation there is a class `ons-details--initialised` set by the JS when JS is on that I was able to use to set print styles on when printing.

This will need a DAC test but I believe this should cover all the current DAC recommendations we have at the moment.

Ive also made the accordion example a bit wider because it was looking a bit narrow and wrapping.

### How to review this PR

- Check details and accordion components look and work as before with JS turned on
- Check details and accordion components now display just a heading and paragraph when printed with JS turned on
- Check details and accordion components now display a heading and paragraph for users with JS turned off
- Check the components still work with screen readers

### Checklist

This needs to be completed by the person raising the PR.
